ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -105,135 +200,92 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "05:59:27  INFO    :         Target - Using base path: /home/derkling/Code/lisa\n",
```

Comment:
Do you guys think that the old logs and configuration that existed in the notebook should be kept or should they be updated, as done here?
I've updated them according to what I obtained running the examples on my machine because I slightly modified the configuration as seen above so I wanted the logs to match that, but to make it consistent I also had to modify the logs for the other examples. But I'm not sure if a few configuration changes are worth updating all run logs.
Let me know what you think.